src/extension/content-script/onend.js

```diff
@@ -31,11 +31,12 @@ async function init() {
   }
 
   injectScript(browser.runtime.getURL("js/inpageScript.bundle.js")); // registers the DOM event listeners and checks webln again (which is also loaded onstart
+  injectBoostButton();
 
   // extract LN data from websites
   browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
     if (request.action === "extractLightningData") {
-      extractLightningData();
+      // extractLightningData();
```

Comment:
Was this intentional or a mistake? If intentional, can't you delete the entire `if` and the event listener then?